### PR TITLE
submitting 2.2 exercise gridprinter, 2.3 exercise fizzbuzz, 2.4 exercise series

### PR DESCRIPTION
**Pull request by nichx313**
PR to submit fizzbuzz exercise

### CODE REVIEW

**Comment by nichx313** [comment]
https://github.com/UWPCE-PythonCert-ClassRepos/SP_Online_PY210/pull/668/commits/f08ef3ba2a1916d165b01e848ce4a0323b80e19f is/was actually fizzbuzz

File: students/nick_miller/lesson03/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+while usrChoose not in fruits:
+    usrChoose = input("Sorry, that fruit is not in the list, enter a fruit to remove from the list: ")
+    usrChoose = usrChoose.strip()
+    usrChoose = usrChoose.capitalize()
+
+while usrChoose in fruits:
+    fruits.remove(usrChoose)
+
+print(fruits)
```
[the remark this answers is not in the document]
Comment (charlesrobison) [reply students/nick_miller/lesson03/list_lab.py]: I don't see this one as submitted on the course site - please submit for grading (or add any missing portions for review)

File: students/nick_miller/lesson03/mailroom.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            report()
+
+        elif usrchoice.strip() == '3':
+            quit_prog()
+
+        else:
+            print("sorry, that's not a valid option")
+
+
+main()
```
[the remark this answers is not in the document]
Comment (charlesrobison) [reply students/nick_miller/lesson03/mailroom.py]: Looks like this needs to be submitted on the course site for grading

**Comment by nichx313** [comment]
Apologies Charles, I’m still getting used to the submission and VCS stuff for the class; this is my first submissions, after passing the wrong link the first time. :facepalm: Fizzbuzz is the one I meant to submit here. I thought the other lesson 2 items were ready, but I’ll backtrack through them and update/re-commit/push before submitting them. Adjusting from the Canvas model of my last class. Should I include the particular commit’s hash for each assignment when I submit? Natasha had said I can do multiple assignments with one PR at least. Thanks for the help!

- Nick Miller

**Comment by charlesrobison** [comment]
You can totally do multiple on one PR.  Just make a note of anything that's not ready for review (or note what is ready, either way).

**Comment by nichx313** [comment]
Sweet, thanks. I appreciate all the feedback on the stuff in there. I somehow completely missed the second portion of the grid_printer exercise. Does this class allow for re-submission on things? Seems less intuitive - Natasha had to delete my first submission so I could put in an actual PR link, as opposed to just a link to my repo. Thanks again!

- Nick Miller

**Comment by charlesrobison** [comment]
Sure, you can just add a re-submission to a PR and make a note that you'd like an other review/grade adjustment.

**Comment by natasha-aleksandrova** [comment]
@nichx313 please note that we do have to manually remove submissions on the course website if they have incorrect links.

**Pull request merged**